css/fonts.css

```diff
@@ -0,0 +1,31 @@
+@font-face {
+  font-family: "MontserratBold";
+  src: url("../fonts/Montserrat-Bold/montserrat-bold-webfont.woff");
+  src: url("../fonts/Montserrat-Bold/Montserrat-Bold.ttf");
+  font-style: normal;
+  font-weight: bold;
+}
+
+@font-face {
+  font-family: "MontserratMedium";
+  src: url("../fonts/Montserrat-Medium/montserrat-medium-webfont.woff");
+  src: url("../fonts/Montserrat-Medium/Montserrat-Medium.ttf");
+  font-style: normal;
+  font-weight: 500;
+}
+
+@font-face {
+  font-family: "MontserratRegular";
+  src: url("../fonts/Montserrat-Regular/montserrat-regular-webfont.woff");
+  src: url("../fonts/Montserrat-Regular/Montserrat-Regular.ttf");
+  font-style: normal;
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: "MontserratSemiBold";
+  src: url("../fonts/Montserrat-SemiBold/montserrat-semibold-webfont.woff");
+  src: url("../fonts/Montserrat-SemiBold/Montserrat-SemiBold.ttf");
+  font-style: normal;
+  font-weight: 600;
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/tesla_project/pullRequest?prid=5520132)